integration/test-integration/src/test/resources/apim/1/apis/BackendSecurityApi.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Shall we keep File names in a common format ? `BackendSecurityApi.json` , `JwtScopeAPI.json` In some of the places API is in capital.

Reply:
Picked `API` instead of `Api` for API names because difference between `API` and `App` is also greater when used within the testcase and in the `subscription.json` file. Fixed with https://github.com/wso2/product-microgateway/pull/2646/commits/6c3651005ec5a97c5b9831526bb82b0b9fce45d9